docker-compose.yml

```diff
@@ -17,7 +17,7 @@ services:
     extends:
       service: base
     ports:
-      - "8000:8000"
+      - "${EXPOSE_ELIOT_PORT:-8000}:8000"
```

Comment:
If you set this in `.env`, then `make run` doesn't really work. Is it possible to set this in `.env` for the local dev environment, but not the devcontainer?

Reply:
> then `make run` doesn't really work

how so? it still runs eliot, just exposed on a different port